src/olympia/landfill/serializers.py

```diff
@@ -64,23 +62,23 @@ def _create_fxa_user(self):
             session.uid == m['headers']['x-uid']
         )
         session.verify_email_code(message['headers']['x-verify-code'])
-        self.fxa_id = session.uid
-        log.debug("fxa account created: {}".format(fxa_account))
+        log.debug('fxa account created: {}'.format(fxa_account))
+        return session.uid
 
     def _create_addon_user(self):
         """Create addon user with fxa information assigned."""
         try:
-            self.user = UserProfile.objects.create_user(
+            return UserProfile.objects.create_user(
                 username='uitest',
                 email=self.fxa_email,
                 fxa_id=self.fxa_id,
                 display_name='uitest'
             )
         except Exception as e:
-            log.debug("There was a problem creating the user: {}."
-                " Returning user from database".format(e)
-            )
-            self.user = UserProfile.objects.get(username='uitest')
+            log.debug('There was a problem creating the user: {}.'
+                      ' Returning user from database'.format(e)
+                      )
```

Comment:
The parenthesis is wrong here, should either be at the beginning of the line or ideally behind the `.format(e)`

Reply:
Flake8 doesn't like it at the beginning.